graphene_django_plus/mutations.py

```diff
@@ -122,17 +122,20 @@ def _get_fields(model, only_fields, exclude_fields, required_fields, registry):
 
         if required_fields is not None:
             required = name in required_fields
-            f.kwargs["required"] = required
+            if hasattr(f, 'kwargs'):
+                f.kwargs["required"] = required
         else:
             if isinstance(field, (ManyToOneRel, ManyToManyRel)):
                 required = not field.null
             else:
                 required = not field.blank and field.default is NOT_PROVIDED
 
-            f.kwargs["required"] = required
+            if hasattr(f, 'kwargs'):
+                f.kwargs["required"] = required
```

Comment:
Could you give a little explanation on why this needed to be done? Because from first glance it seems that `required` might be set only when kwargs are defined, but probably there might be cases when kwargs are not defined, but `required` might be still set... but that's just IMHO

Reply:
I was getting `AttributeError` for out tests. And added this check as easy fix, but not sure if this is best solution.
```
tests/test_utils.py:8: in <module>
    from .base import BaseTestCase
tests/base.py:9: in <module>
    from .schema import schema
tests/schema.py:116: in <module>
    class IssueCreateMutation(ModelCreateMutation):
../../Library/Caches/pypoetry/virtualenvs/graphene-django-plus-6Kh6sXUW-py3.10/lib/python3.10/site-packages/graphene/types/objecttype.py:29: in __new__
    base_cls = super().__new__(
../../Library/Caches/pypoetry/virtualenvs/graphene-django-plus-6Kh6sXUW-py3.10/lib/python3.10/site-packages/graphene/utils/subclass_with_meta.py:46: in __init_subclass__
    super_class.__init_subclass_with_meta__(**options)
graphene_django_plus/mutations.py:697: in __init_subclass_with_meta__
    super().__init_subclass_with_meta__(
graphene_django_plus/mutations.py:409: in __init_subclass_with_meta__
    fdata = _get_fields(model, only_fields, exclude_fields, required_fields, registry)
graphene_django_plus/mutations.py:139: in _get_fields
    required = f.kwargs["required"]
E   AttributeError: 'BlankValueField' object has no attribute 'kwargs'
```

Reply:
Hmm, pretty strange why kwargs has been removed... Have you tried taking a look why it was removed? Maybe there is an alternative starting with new graphene version?